app/models/user/user.go

```diff
@@ -62,13 +62,18 @@ func Find(username string) *User {
 	res, err := r.Table("users").Filter(map[string]string{
 		"username": username,
 	}).Run(db.Session)
-	if err != nil || res.IsNil() {
-		return nil
+
+
+	var row *User
+	err2 := res.One(&row)
+	if err2 == r.ErrEmptyResult || err != nil || err2 != nil {
+	    return nil
 	}
-	var user *User
-	return res.One(&user)
+
+	return row
```

Comment:
I think you should name this user instead of row.  Calling it `row` isn't entirely clear, and the purpose of the function is to return a user.  We may also want to refactor this function to return `*User, error`.